### PR TITLE
DIV-4587 - Accessibility [AA]: Ambiguous links

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-4587 - Accessibility [AA]: Ambiguous links](https://tools.hmcts.net/jira/browse/DIV-4587)
 - Add aria-label to each link

### CODE REVIEW

File: app/steps/done-and-submitted/content.json [hunk 1]
```diff
@@ -27,17 +27,17 @@
           "youCanFindMore": "Your {{divorceWho}} will also receive a copy of the application and a form to return. If you have used adultery as your reason for divorce, and named the person that your {{divorceWho}} committed adultery with, they will also get a copy and a form to return.",
           "contactTheCourt": "Contact the divorce centre if you don't hear anything back after 3 weeks.",
           "getHelp": "If you need help",
-          "getHelpDescription": "You can contact the divorce centre if you need help with your application. Court staff can’t give you legal advice however, and you should speak to a <a class=\"govuk-link\" href=\"https://www.gov.uk/find-a-legal-adviser\" target=\"_blank\">solicitor or legal adviser</a> if you need help with that.",
+          "getHelpDescription": "You can contact the divorce centre if you need help with your application. Court staff can’t give you legal advice however, and you should speak to a <a class=\"govuk-link\" href=\"https://www.gov.uk/find-a-legal-adviser\" target=\"_blank\" aria-label=\"Find a leagal adviser link, This will open a new tab.\">solicitor or legal adviser</a> if you need help with that.",
```
Comment (hosainnet) [inline app/steps/done-and-submitted/content.json]: `leagal` typo

File: app/steps/marriage/upload/content.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
...
-                    "marriageCertificateContent": "your original marriage certificate or a <a class=\"govuk-link\" href=\"https://www.gov.uk/order-copy-birth-death-marriage-certificate\" target=\"_blank\">certified copy</a>",
-                    "translationContent" : "a <a class=\"govuk-link\" href=\"https://www.gov.uk/certifying-a-document\" target=\"_blank\">certified English translation</a> of your marriage certificate",
-                    "nameChangeContent" : "proof of your name change, for example a <a class=\"govuk-link\" href=\"https://www.gov.uk/change-name-deed-poll/overview\" target=\"_blank\">deed poll</a>. Upload proof of each change if you’ve done it more than once since you got married",
+                    "marriageCertificateContent": "your original marriage certificate or a <a class=\"govuk-link\" href=\"https://www.gov.uk/order-copy-birth-death-marriage-certificate\" target=\"_blank\" aria-label=\"Certified copy link, This will open a new tab. You'll need to return to this tab and continue with your application within 60 mins so you don’t lose your progress.\">certified copy</a>",
```
Comment (hosainnet) [inline app/steps/marriage/upload/content.json]: Should it be `60 minutes` if it's a screen reader thing? (no idea how it'll read it out)

Comment (dtaylor7) [reply app/steps/marriage/upload/content.json]: 👍

File: app/steps/done-and-submitted/content.json (hunk 1 above)
Comment (dtaylor7) [reply app/steps/done-and-submitted/content.json]: 👍

**Pull request merged**